java/ingest/src/test/java/sleeper/ingest/job/IngestJobRunnerIT.java

```diff
@@ -269,13 +253,11 @@ public void shouldBeAbleToHandleAllFileFormatsPutOnTheQueue() throws Exception {
         FileSystem.get(uri2, AWS_EXTERNAL_RESOURCE.getHadoopConfiguration()).createNewFile(new Path(uri2));
         files.add(getIngestBucket() + "/file-2.csv");
         IngestJob ingestJob = new IngestJob(TEST_TABLE_NAME, "id", files);
-        AWS_EXTERNAL_RESOURCE.getSqsClient()
-                .sendMessage(getInstanceProperties().get(INGEST_JOB_QUEUE_URL), new IngestJobSerDe().toJson(ingestJob));
-        consumeAndVerify(recordListAndSchema.sleeperSchema, recordListAndSchema.recordList, 1);
+        consumeAndVerify(recordListAndSchema.sleeperSchema, ingestJob, recordListAndSchema.recordList, 1);
     }
 
     @Test
-    public void shouldIngestParquetFilesInNestedDirectoriesPutOnTheQueue() throws Exception {
+    public void shouldIngestParquetFilesInNestedDirectories() throws Exception {
```

Comment:
Can we change the numbers in this test to make it faster without affecting the value of the test? This seems to go from 40 seconds for each invocation that uses S3, to 40 seconds overall:

```java
        RecordGenerator.RecordListAndSchema recordListAndSchema = RecordGenerator.genericKey1D(
                new LongType(),
                LongStream.range(-5, 5).boxed().collect(Collectors.toList()));
        int noOfTopLevelDirectories = 2;
        int noOfNestings = 2;
        int noOfFilesPerDirectory = 2;
```

Reply:
For me this goes from 2 minutes to 45 seconds, will include this change.

I tried changing the range for the other tests but it didn't affect the test runtime too much.